### ISSUE TITLE
Word wrap code

### ISSUE THREAD

**Issue by bigfish24** [GitHawkApp/GitHawk#1084]
Due to screen size it would be great to have option to word wrap PR code text views.

**Comment by rnystrom** [comment]
So we used to have it, but removed to be more like GitHub. I like the idea of making it an _option_ tho, that’s really cool.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Sherlouk** [comment]
@rnystrom Might be worth trying to get #577 moved forward (or a similar PR) as it'd get the ball moving on functionality like this and review comments etc! The code view needs quite the number of improvements before we can get all the nice features we want!

**Comment by BasThomas** [comment]
Closing this in favor of #2599
